serialization.md

```diff
@@ -91,6 +91,8 @@ Here is an example image JSON file:
 
 ```
 {  
+    "schemaVersion": 2,
```

Comment:
Honestly I'm unclear that it applies here, but there ought to be something like this. And more well defined in the `serialization.md`